addons/planner_project/data/planner_data.xml

```diff
@@ -0,0 +1,1211 @@
+                                    </p>
+                                </div>
+                                <div class="col-sm-5 col-md-5">
+                                    <img src="/planner_project/static/src/img/task.png" />
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-sm-1 col-md-2 icon"><span data-icon="&#xe075;" /></div>
+                                <div class="col-sm-11 col-md-10">
+                                    <p><strong>2. Add a Deadline</strong><br/>
+                                        The deadline will help you determine if a task or issue is progressing as expected, and to anticipate its next update.
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-sm-1 col-md-2 icon"><span data-icon="&#xe055;" /></div>
+                                <div class="col-sm-11 col-md-10">
+                                    <p><strong>3. Use Tags</strong><br/>
+                                        Tags  are complementary to your project stages: they can work as a second  level categorization which is very useful if you have a lot of Tasks or Issues to manage. Also, it becomes very easy to find Tasks or Issues by typing  the Tag into the main Search bar.
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-sm-1 col-md-2 icon"><span data-icon="&#xe007;" /></div>
+                                <div class="col-sm-11 col-md-10">
+                                    <p><strong>4. Use Status Indicator</strong><br/>
+                                        The Status indicator helps you manage Tasks and Issues by attributing them one of 3 different colour: Grey, Green or Red. The meaning of these Status can be freely configured, for example:
+                                        <li>Grey: the Task is in progress (someone is working on it)</li>
+                                        <li>Green: the Task is ready for next stage (the job for this stage is complete)</li>
+                                        <li>Red: the Task is blocked (there's a problem)</li></p>
+                                    <p>To configure these Kanban Status, got to the 'Project Stages' and click on a Stage.</p>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col-sm-1 col-md-2 icon"><span data-icon="&#xe017;" /></div>
```

Comment:
icon not well rendered on chrome Version 40.0.2214.115 (64-bit). I see a square instead :crying_cat_face: